src/main/java/com/microsoft/sqlserver/jdbc/IOBuffer.java

```diff
@@ -749,17 +752,20 @@ void resetPooledConnection() {
 
     /**
      * Opens the physical communications channel (TCP/IP socket and I/O streams) to the SQL Server.
+     * 
+     * @param IPAddressPreference
```

Comment:
Starting on line 2922 of this file, you add descriptions for your params, but not here. It should be one or the other (I would assume we would prefer to have descriptions for all params).